lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -220,100 +202,6 @@ std::string _GenerateXMLString(const HdMaterialNetwork& materialNetwork, bool in
     return result;
 }
 
-//! Helper utility function to apply VP2-specific fixes to the material network.
-//! - Add passthrough nodes to read vector component(s).
-//! - Fix UsdImagingMaterialAdapter issue for not producing primvar requirements.
-//! - Temporary workaround of missing support for normal map.
-void _ApplyVP2Fixes(HdMaterialNetwork& outNet, const HdMaterialNetwork& inNet)
```

Comment:
The method is moved down into the class section because it is now a member of HdVP2Material. I need to do so for accessing new member of the class.